snap/snapcraft.yaml

```diff
@@ -112,39 +126,19 @@ parts:
         cp -r jars/* $CRAFT_PART_INSTALL/jars/
         mkdir -p $CRAFT_PART_INSTALL/python
         cp -r python/* $CRAFT_PART_INSTALL/python/
+
     override-prime: |
         snapcraftctl prime
         rm -vf usr/lib/jvm/java-11-openjdk-*/lib/security/blacklisted.certs
 
   helper-package:
-    plugin: python
```

Comment:
**suggestion** What about changing the name from `helper-package` to `kubectl`?

Reply:
Makes sense, done :-)